cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -43,6 +45,11 @@ var Cmd = &cobra.Command{
 			return err
 		}
 
+		store, err := helper.GetBool("store")
```

Comment:
The storage of results hasn't proven very useful. IMO we can remove that feature.

Reply:
yes will remove them

Reply:
removed